UPGRADING

```diff
@@ -21,6 +21,8 @@ PHP 7.1 UPGRADE NOTES
 - Core:
   . 'void' can no longer be used as the name of a class, interface, or trait.
     This applies to declarations, class_alias() and use statements.
+  . The ASCII 0x7F Delete/Backspace control character is no longer permitted
+    in unquoted identifiers in source code.
```

Comment:
I hope "unquoted identifiers" is reasonably clear, but I'm open to suggestions for better wording. I want to express that it covers `$a␡b` and `a␡b` but not `${"a\x7Fb"}` or `constant("a\x7Fb")`.